src/main/kotlin/stringcalculator/Operator.kt

```diff
@@ -0,0 +1,30 @@
+package stringcalculator
+
+interface Operator {
+    fun apply(num1: Int, num2: Int): Int
+}
+
+class Adder : Operator {
+    override fun apply(num1: Int, num2: Int): Int {
+        return num1 + num2
+    }
+}
+
+class Subtractor : Operator {
+    override fun apply(num1: Int, num2: Int): Int {
+        return num1 - num2
+    }
+}
+
+class Multiplier : Operator {
+    override fun apply(num1: Int, num2: Int): Int {
+        return num1 * num2
+    }
+}
+
+class Divider : Operator {
+    override fun apply(num1: Int, num2: Int): Int {
+        if (num2 == 0) throw IllegalArgumentException("0으로 나눌 수 없습니다.")
+        return num1 / num2
```

Comment:
```suggestion
        return if (num2 == 0) {
            throw IllegalArgumentException("0으로 나눌 수 없습니다.")
       } else {
           num1 / num2
       }
```

이런식으로도 표현할 수 있습니다. 개인적으로는 분기문을 포함하여 블럭`{}` 생략을 하는 부분에 대해서 지양하자고 하는편인데요. 
간단한 한줄짜리 블럭이니 없애는게 더 가독성이 높아질수도 있겠지만, 전체적으로 코드의 일관성을 깨트리기 때문에 좀 엄격하게 컨벤션을 통일시키는편인데 좌훈님은 어떻게 생각하시나요?

Reply:
아하 코드 일관성 측면에서는 그렇게 생각할 수도 있겠네요! 이렇게 쓰는게 좀 더 가독성이 좋아보였는데 컨벤션이 있다면 따르는 것도 좋은 방법인 것 같습니다! 요것은 반영해두겠습니다 감사합니다.